go/libkb/burstlimit.go

```diff
@@ -0,0 +1,64 @@
+package libkb
+
+import "time"
+
+// BurstLimiter is a rate limiter that allows bursts of requests.
+type BurstLimiter struct {
```

Comment:
did Go's rate limiter not cut it?

Reply:
It seemed like overkill and tuned for much speedier rates.